README.md

```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
+|  | [Windows containers](https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/) <br> ([kubernetes releases](https://github.com/kubernetes/kubernetes/releases)) <br> [Tests](https://testgrid.k8s.io/sig-windows#aks-engine-azure-1-17-windows) | Run Windows server containers with Kubernetes | CNCF: stable <br> API: N/A | [#sig-windows](https://kubernetes.slack.com/archives/C0SJ4AFB7) <br> [kubernetes-sig-windows@googlegroups.com](https://groups.google.com/forum/#!forum/kubernetes-sig-windows) <br> [Windows Community Forum](https://discuss.kubernetes.io/c/general-discussions/windows) <br> [GitHub issues](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+label%3Asig%2Fwindows+) | [AKS Windows](https://docs.microsoft.com/en-us/azure/aks/windows-container-cli) <br> [AKS Engine Windows](https://github.com/Azure/aks-engine/blob/master/docs/topics/windows.md) |
+|  | [IPv4/v6 Dual Stack](https://kubernetes.io/docs/concepts/services-networking/dual-stack/) <br> ([kubernetes releases](https://github.com/kubernetes/kubernetes/releases)) <br> [Tests](https://testgrid.k8s.io/provider-azure-dualstack) | IPv4/IPv6 dual-stack enables the allocation of both IPv4 and IPv6 addresses to Pods and Services. | CNCF: stable <br> Feature: alpha | [#sig-network](https://kubernetes.slack.com/archives/C09QYUH5W) <br> [kubernetes-sig-network@googlegroups.com](https://groups.google.com/forum/#!forum/kubernetes-sig-network) <br> [GitHub issues](https://github.com/kubernetes/kubernetes/labels/area%2Fipv6) | [Use with AKS Engine](https://github.com/Azure/aks-engine/tree/master/examples/dualstack) |
+| Cloud Native Governance and Security |  |  |  |  |
+|  | [AAD Pod Identity](https://github.com/Azure/aad-pod-identity) <br> ([releases](https://github.com/Azure/aad-pod-identity/releases)) | Enables K8s applications to access cloud resources securely with Azure Active Directory | Azure: incubation <br> API: v1 | [GitHub issues](https://github.com/Azure/aad-pod-identity/issues) <br> [GitHub Project](https://github.com/Azure/aad-pod-identity/projects/3) | [Use with AKS](https://docs.microsoft.com/en-us/azure/aks/developer-best-practices-pod-security#use-pod-managed-identities) <br> [Use with AKS Engine](https://github.com/Azure/aks-engine/tree/master/examples/addons/aad-pod-identity) |
+|  | [OPA Gatekeeper](https://github.com/open-policy-agent/gatekeeper) <br> ([releases](https://github.com/open-policy-agent/gatekeeper/releases)) | K8s native Open Policy Agent policy enforcement | CNCF: incubating <br> API: alpha | [#kubernetes-policy](https://openpolicyagent.slack.com/archives/CDTN970AX) <br> [GitHub issues](https://github.com/open-policy-agent/gatekeeper/issues) | [Azure Policy for AKS](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/rego-for-aks) <br> [Azure Policy for AKS Engine](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/aks-engine) |
```

Comment:
API: v1beta1 and v1alpha1 (there are multiple resources)

Reply:
We should also add `Azure Policy for Azure Arc connected clusters`. Once the doc is published, we can add the link later.

Reply:
Ack